chapters/ch12/0.setup.ipynb

```diff
@@ -116,10 +116,37 @@
       " |-- upc: long (nullable = true)\n",
       " |-- name: string (nullable = true)\n",
       " |-- manufacturer: string (nullable = true)\n",
-      " |-- long_description: string (nullable = true)\n",
       " |-- short_description: string (nullable = true)\n",
+      " |-- long_description: string (nullable = true)\n",
       "\n",
-      "Successfully written 48204 documents\n"
+      "Successfully written 48194 documents\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "----------------------------------------\n",
```

Comment:
@dcrouch26 - output in this notebook has errors. Please re-run to generate clean output (and ensure it matches book output).